openfisca_web_api/wsgihelpers.py

```diff
@@ -96,10 +96,13 @@ def respond_json(ctx, data, code = None, headers = None, jsonp = None):
                 for name, value in error.iteritems()
                 if value is not None
                 )
+        if isinstance(error, str):
+            error = data['error'] = dict(
+                message = data['error'],
+                code = 500
```

Comment:
I don't think it is a good idea to hardcode 500, the `code` argument of `respond_json` should be used, without setting a default value for the `code` argument (`code = code or 500`).

But we prefer to be explicit in the caller, so I don't like this part of the patch.
I would prefer to add an assertion, replacing:

```
if isinstance(error, collections.Mapping):
```

by

```
if error is not None:
    assert isinstance(error, collections.Mapping) and error.get('code') is not None
    data = [...etc...]
```